site/hero.php/index.php

```diff
@@ -0,0 +1,11 @@
+<?php template("hero.php", array('text' => 'Example Page')) ?>
```

Comment:
Hey @junqiangmai , you don't want this line of code here. Basically you're including this file in itself which will cause an infinite loop. You want to use this line of code inside your `index.php` file.